### PR TITLE
delete redundance import

### PR DESCRIPTION
**Pull request by Hazlank**
fuck you

### CODE REVIEW

**Comment by Nbsaw** [comment]
thank and fuck u

File: src/sass/main.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+@import "./common/var",
+"mixin/bem",
+"mixin/helper";;
```
Comment (Nbsaw) [inline src/sass/main.scss]: nmb , 为什么多了个分号

Comment (Hazlank) [reply src/sass/main.scss]: it is none of your bussiness，i like it

Comment (Hazlank) [reply src/sass/main.scss]: It can use v8  engine to accelerate code resolve，do you like？

**Pull request merged**